sdk/src/types/api/core.rs

```diff
@@ -356,11 +356,13 @@ pub enum BlockState {
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub enum TransactionState {
-    // Stored but not confirmed.
+    // Not included yet.
     Pending,
-    // Confirmed with the first level of knowledge.
+    // Included.
+    Accepted,
+    // Included and its included block is confirmed.
     Confirmed,
-    // Included and can no longer be reverted.
+    // Included, its included block is finalized and cannot be reverted anymore.
```

Comment:
```suggestion
    // Included and its confirmed block is finalized (cannot be reverted anymore).
```
at this point the block is confirmed, right?

Reply:
The block is finalized :trollface: Don't think confirmed is right there, I'd say included is also not very clear, but I just copied like this from the TIP

Reply:
ah, I think you're right ... finalized can also mean definitely not confirmed and never going to be, right?

Reply:
It's first confirmed and then it gets finalized and when it's finalized the state can never change anymore